sympy/printing/tests/test_latex.py

```diff
@@ -1958,3 +1958,16 @@ def test_imaginary_unit():
     assert latex(1 + I, imaginary_unit='foo') == '1 + foo'
     assert latex(I, imaginary_unit="ti") == '\\text{i}'
     assert latex(I, imaginary_unit="tj") == '\\text{j}'
+
+
+def test_DiffGeomMethods():
+    from sympy.diffgeom import Manifold, Patch, CoordSystem, Point, BaseScalarField
+    r0, theta0 = symbols('r0, theta0')
```

Comment:
These were apparently used later on in the code snippet I copied from... Will remove them, but waiting for review comments first.